packages/SwingSet/test/test-devices.js

```diff
@@ -1,7 +1,8 @@
 import { test } from 'tape-promise/tape';
 import harden from '@agoric/harden';
+import { makeMemorySwingStore, getAllState } from '@agoric/swing-store-simple';
```

Comment:
I'm on the fence about whether `getAllState` should be a separate function (which takes a `storage` instance), or a method on the `storage` instance. The former is wrong because the authority to read out the contents should come from the storage object, not some stranger who can peek inside. The latter is wrong because the kernel shouldn't get the read-all-state authority (well, it has that authority, but it doesn't need it in this weird debugging form that returns an object rather than a key-reading API).

Using a separate function is probably the best way to go, but I figured I should bring it up.

Reply:
getAllState as written has no authority; it is implemented entirely in terms of the storage API.  It's just there to avoid repeating code.  I want to keep it separate because it's only for testing and debugging and shouldn't be used in general.